config/environments/development.rb

```diff
@@ -110,8 +110,8 @@
   # Raise error when a before_action's only/except options reference missing actions
   config.action_controller.raise_on_missing_callback_actions = true
 
-  # Send mails to browser window
-  config.action_mailer.delivery_method = :letter_opener_web
+  # Send mails to browser window (unless configured otherwise)
+  config.action_mailer.delivery_method = ENV["OPENPROJECT_EMAIL_DELIVERY_METHOD"]&.to_sym || :letter_opener_web
```

Comment:
I wonder if this could be `OpenProject::Configuration`, but it should be fine for dev purposes